security/rbac/scripts/enable-rbac-ksql-server.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+
+################################################################################
+# Overview
+################################################################################
+#
+################################################################################
+
+# Source library
+. ../../../utils/helper.sh
+. ./rbac_lib.sh
+
+check_env || exit 1
+check_cli_v2 || exit 1
+check_jq || exit 1
+
+##################################################
+# Initialize
+##################################################
+
+. ../config/local-demo.cfg
+ORIGINAL_CONFIGS_DIR=/tmp/original_configs
+DELTA_CONFIGS_DIR=../delta_configs
+FILENAME=ksql-server.properties
+create_temp_configs $CONFLUENT_HOME/etc/ksql/$FILENAME $ORIGINAL_CONFIGS_DIR/$FILENAME $DELTA_CONFIGS_DIR/${FILENAME}.delta
+
+# Log in to Metadata Server (MDS)
+login_mds $MDS
+
+##################################################
+# Administrative Functions
+# - Grant principal User:$ADMIN_KSQL the ResourceOwner role to Topic:_confluent-ksql-${KSQL_SERVICE_ID}_command_topic
+# - Grant principal User:$ADMIN_KSQL the ResourceOwner role to Topic:${KSQL_SERVICE_ID}_ksql_processing_log
+# - Start KSQL
+# - Grant principal User:$ADMIN_KSQL the SecurityAdmin role to the KSQL Cluster
```

Comment:
should this be done before starting ksql?

Reply:
It doesn't seem to be required before KSQL server start, which is why I deliberately put it after to draw attention to this.